terracumber_config/tf_files/SUSEManager-Test-Hexagon-NUE.tf

```diff
@@ -132,30 +131,34 @@ module "cucumber_testsuite" {
       }
     }
     server_containerized = {
+      image = "slemicro55o"
       provider_settings = {
         mac = "aa:b2:93:01:00:51"
-        memory = 16384
+        vcpu = 8
+        memory = 32768
       }
       login_timeout = 28800
       runtime = "podman"
-      container_repository = "registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni"
-      //additional_repos = {
-      //  Test_repo = "http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/TEST:/Hexagon/SLE_15_SP4/"
-      //}
-    }
-    proxy = {
-      provider_settings = {
-        mac = "aa:b2:93:01:00:52"
+      container_repository = "registry.suse.de/devel/galaxy/manager/test/hexagon/containerfile/suse/manager/5.0/x86_64"
+
+      additional_repos = {
+        systemsmanagement_Uyuni_Master_ServerContainer = "https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/ServerContainer/SLE-Micro55/"
```

Comment:
Why don't you use TEST:Hexagon here as it's for SUMA?

Reply:
I did it for not creating another pipeline. Currently we have `manager-TEST-Hexagon-2obs-uyuni-tools` that push in OBS, instead for HEAD we have `manager-Head-uyunitools-2obs` that  push in IBS (different from the one used for OBS). But yes, you are right, it might be better to change `manager-TEST-Hexagon-2obs-uyuni-tools` to push in `TEST:Hexagon`. I'm going to change it